func/main.sh

```diff
@@ -403,8 +403,8 @@ parse_object_kv_list() {
 
 # Check if object is supended
 is_object_suspended() {
-	if [ $2 = 'USER' ]; then
-		spnd=$(cat $USER_DATA/$1.conf | grep "SUSPENDED='yes'")
+	if [ "$2" = 'USER' ]; then
+		spnd=$(grep "SUSPENDED='yes'" | $USER_DATA/$1.conf)
```

Comment:
this doesn't look right, i think you meant
```
		spnd=$(grep "SUSPENDED='yes'" "$USER_DATA/$1.conf")
```

Reply:
Has been fixed